ballerina/tests/README.md

```diff
@@ -1,6 +1,41 @@
-## Prerequisites for Running Tests
+# Running Tests
 
-To run the tests, you need Twilio account credentials,twilio phone number and test phone number. To get Twilio account credentials and twilio phone number;
+There are two test environments for the Twilio connector. The default test environment is the mock server for Twilio API. The other test environment is the actual Twilio API. You can run the tests in either of these environments and each has its own compatible set of tests.
+
+Test Groups | Environment
+---| ---
+mock_tests | Mock server for Twilio API (Defualt Environment)
+actual_tests | Twilio API
```

Comment:
```suggestion
prod_tests | Twilio API
```
Rather than `actual` shall we use `prod` or `live` ? I think we should reconsider the configurable/environmental variable naming as well.

Reply:
+1

Reply:
Sure, changed with [5209838](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/218/commits/520983883c21405cb6f00e7c862a39526df6246d).